### DISCUSSION TITLE
How to clean the music genres list?

### DISCUSSION THREAD

**Discussion by DameGAD** [navidrome/navidrome#3600] [Q&A]
I recently retagged some files. In the process, I completely removed a list of genre tags.

The removed genres are correctly removed from the release pages in Navidrome's UI, but some of them are still displayed in the genre filter list (not all of them, some were also removed from there). Is there a way to force Navidrome to remove them?

Here is an example with Other. It is not used by any file anymore, but it is still listed in the UI.

![Screenshot 2024-12-27 à 16 11 14](https://github.com/user-attachments/assets/8c25aa88-e00b-41ab-85e4-cc7c74486591)

**Comment by deluan** [comment]
Not sure what is the issue here and how this got out of sync, but in the next release we will introduce a lot of changes in the DB layout and the scanner (see PR #2709). making sure things stay in sync is one of the goals of the new scanner.
